modules/vault-app-policy/main.tf

```diff
@@ -0,0 +1,31 @@
+data "template_file" "app_readonly" {
+  template = "${file("${path.module}/templates/kv_app.hcl")}"
+
+  vars {
+    kv_path = "${var.kv_path}/app/${var.app}"
+  }
+}
+
+resource "vault_policy" "app_readonly" {
+  name   = "${local.app_policy_name}"
+  policy = "${data.template_file.app_readonly.rendered}"
+}
+
+data "template_file" "app_read_write" {
+  template = "${file("${path.module}/templates/kv_dev.hcl")}"
+
+  vars {
+    kv_path = "${var.kv_path}/app/${var.app}"
+  }
+}
+
+resource "vault_policy" "app_read_write" {
+  name   = "${local.dev_policy_name}"
+  policy = "${data.template_file.app_read_write.rendered}"
+}
+
+locals {
```

Comment:
Could use [`join`](https://www.terraform.io/docs/configuration/interpolation.html#join-delim-list-). Then just play around with the delim.

Reply:
Fixed, thanks for the hint